extension/src/popup/components/sendPayment/SendSettings/index.tsx

```diff
@@ -98,9 +98,9 @@ export const SendSettings = ({
       );
 
       const params = [
-        new SorobanClient.Address(publicKey).toScVal(), // from
-        new SorobanClient.Address(destination).toScVal(), // to
-        new SorobanClient.XdrLargeInt("i128", parsedAmount.toNumber()).toI128(), // amount
+        new Address(publicKey).toScVal(), // from
+        new Address(destination).toScVal(), // to
+        new XdrLargeInt("i128", parsedAmount.toNumber()).toI128(), // amount
```

Comment:
`new ScInt(parsedAmount).toI128()` is equivalent/shorter :shrug:

Reply:
Thanks!